api/src/nv_ingest_api/internal/schemas/store/store_image_schema.py

```diff
@@ -17,14 +17,29 @@
 # limitations under the License.
 
 import logging
+import os
+from typing import Optional, Dict, Any
 
-from pydantic import ConfigDict, BaseModel
+from pydantic import ConfigDict, BaseModel, Field, field_validator
 
 logger = logging.getLogger(__name__)
 
 
+_DEFAULT_STORAGE_URI = os.environ.get("IMAGE_STORAGE_URI", "s3://nv-ingest/artifacts/store/images")
+
+
 class ImageStorageModuleSchema(BaseModel):
     structured: bool = True
     images: bool = True
+    storage_uri: str = Field(default_factory=lambda: _DEFAULT_STORAGE_URI)
+    storage_options: Dict[str, Any] = Field(default_factory=dict)
+    public_base_url: Optional[str] = None
```

Comment:
How does this play in, couldn't I get everything I need from the storage_uri? If not, in what cases doesnt that work?